scene/main/scene_tree.cpp

```diff
@@ -1519,10 +1522,30 @@ void SceneTree::set_multiplayer(Ref<MultiplayerAPI> p_multiplayer, const NodePat
 	} else {
 		if (custom_multiplayers.has(p_root_path)) {
 			custom_multiplayers[p_root_path]->object_configuration_remove(nullptr, p_root_path);
+		} else {
+			const Vector<StringName> tnames = p_root_path.get_names();
+			const StringName *nptr = tnames.ptr();
+			for (const KeyValue<NodePath, Ref<MultiplayerAPI>> &E : custom_multiplayers) {
+				const Vector<StringName> snames = E.key.get_names();
+				if (tnames.size() < snames.size()) {
+					continue;
+				}
+				const StringName *sptr = snames.ptr();
+				bool valid = true;
+				for (int i = 0; i < snames.size(); i++) {
+					if (sptr[i] != nptr[i]) {
+						valid = false;
+						break;
+					}
+				}
+				ERR_FAIL_COND_MSG(valid, "Multiplayer is already configured for a parent of this path: '" + p_root_path + "' in '" + E.key + "'.");
+			}
```

Comment:
I think this check applies only when `p_multiplayer.is_valid()`.
I know this is pretty nested already, maybe we can refactor the outer ifs as an alternative? In any case, I'd avoid failing during removal.

Reply:
How about this?

Reply:
Wait one moment

Reply:
Oh that's nice!